cmd/otel-allocator/server/bench_test.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func BenchmarkServerTargetsHandler(b *testing.B) {
-	rand.Seed(time.Now().UnixNano())
+	random := rand.New(rand.NewSource(time.Now().UnixNano())) // nolint: gosec
```

Comment:
Adding gosec to avoid 

```
server/bench_test.go:34:2: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
```